### DISCUSSION TITLE
Les pays

### DISCUSSION THREAD

**Discussion by evefevrier** [integration2/quizTIM2023#49] [Q&A]
Commentaires sur ce quiz.

**Accès**
- Serveur de développement (protégé) :  https://timunix3.csfoy.ca/~quiz-TIM/lauriem1/index.html
- Dépôt Github (privé pour le moment): https://github.com/lauriem1/quiz-tim
- Github.io :

**Comment by Eli7792** [comment]
Logo et couleur de fond originale, cependant, il semble y avoir un souci au niveau du javascript. Il est impossible de cliquer sur les boutons et de valider son choix.
